releasenotes/notes/user-tag-interface-metrics-597f442a7de2787b.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Add a new tag to interface metrics that will allow them to be tagged by user tags.
```

Comment:
```suggestion
    Add a new tag to interface metrics that will allow them to be tagged by user-defined tags.
```

Reply:
As an alternative, I would also suggest considering something along the lines of: 

"Add the ability to tag interface metrics with user-defined tags."

You mention that the PR adds an internal tag, so maybe users don't need to know that specific detail.

Reply:
updated eb6f55bc31088f140c2f47aebe8750bc5d99140b